ray-curator/ray_curator/models/pos_embed.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) OpenGVLab. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
```

Comment:
Maybe we rename this file to utils or something? Is this generic enough for other models too or if it's specific I2V, we can move it there?